public/app/frontend/src/components/search-bar/search-bar.html.twig

```diff
@@ -0,0 +1,46 @@
+{#
+@file
+A search bar
+
+Available variables:
+    - action: string The form action
+    - id: string The unique id of the form element
+    - input: object The variables for the input component
+    - button: object The variables for the button component
+example:
+    {% include '@components/search-bar/search-bar.html.twig' with {
+        action: '/',
+        id: 'search-top'
+        input: {
+            labelHidden: true,
+            label: 'Search',
+            id: 'search-top-input',
+        },
+        button: {
+            style: 'primary',
```

Comment:
Super minor, as style is sometimes used in jsx as css. e.g. `style={{'padding':'10px'}}`.

Could we use `variant: 'primary'` or something else other than style?

Reply:
Good shout! I've updated that :)